scanner/src/funTest/assets/analyzer-result.yml

```diff
@@ -15,6 +15,7 @@ analyzer:
   start_time: "1970-01-01T00:00:00Z"
   end_time: "1970-01-01T00:00:00Z"
   environment:
+    ort_version: "9c6ed1c"
```

Comment:
Why do we have different versions on the test files (`72463cc`, `9c6ed1c`, `HEAD`)?

Reply:
Where possible, I used `git blame` to find out with which version of ORT a particular `environment` entry for a run was created and have used the abbreviated SHA1 for that commit.

In cases where we would otherwise need to patch the *expected* result with the current version of ORT, I decided to instead use `HEAD` in the expected result and patch the *actual* result so we do not have to query the VCS working tree in `patchExpectedResult`.

Reply:
Thanks for the explanation.